eea/src/main/java/org/web3j/protocol/eea/response/PrivateTransaction.java

```diff
@@ -0,0 +1,155 @@
+package org.web3j.protocol.eea.response;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+@JsonDeserialize(using = PrivateTransaction.ResponseDeserialiser.class)
+public abstract class PrivateTransaction {
```

Comment:
Saw that PrivateTransactionReceipt extends TransactionReceipt - is there any reason why this doesn't extend the usual Transaction class?

Reply:
No, because the Transaction class contains fields such as `blockHash` and `transactionIndex` which are not in `PrivateTransaction`